website/docs/usage.md

```diff
@@ -34,9 +68,28 @@ import Spreadsheet from "react-spreadsheet";
 
 const App = () => {
   const [data, setData] = useState([
-    [{ value: "Vanilla" }, { value: "Chocolate" }],
-    [{ value: "Strawberry" }, { value: "Cookies" }],
+    [{ value: "Vanilla" }, { value: "Chocolate" }, { value: "" }],
+    [{ value: "Strawberry" }, { value: "Cookies" }, { value: "" }],
   ]);
-  return <Spreadsheet data={data} onChange={setData} />;
+  return <Spreadsheet data={data} 
+  onChange={
+    const t = d
+    for (let i = 0; i < 3; i++) {
+      const string = d[i][1]
+      if(string.value === 'Chocolate'){
+        t[i][2] = {value:"🍫"}
+      }
+      else if(string.value === "Cookies"){
+        t[i][2] = {value:"🍪"}
+      }
+      else if(string.value === "Pizza"){
+        t[i][2] = {value:"🍕"}
+      }
+      else {
+        t[i][2] = {value:"❓"}
+      }
+      setData(t)
+    }
```

Comment:
In react it's recommended not to mutate source data. Make sure to duplicate the array before mutating it.

Reply:
okay I'll make the required changes.